core/templates/domain/skill/concept-card-backend-api.service.ts

```diff
@@ -89,7 +89,7 @@ export class ConceptCardBackendApiService {
     return uncachedSkillIds;
   }
 
-  loadConceptCards(skillIds: string[]): Promise<ConceptCard[]> {
+  async loadConceptCardsAsync(skillIds: string[]): Promise<ConceptCard[]> {
```

Comment:
@Deepam101 I was expecting this function to be used somewhere in another directive/services but I don't see one. Can you please check whether this function is used anywhere?

Reply:
There were a couple of files using `loadConceptCards` which I changed to `loadConceptCardsAsync`